example/pages/users/index.tsx

```diff
@@ -41,25 +47,32 @@ function Users({ usersQuery }) {
   );
 }
 
-function UsersWrapper(props: any) {
+function UsersWrapper(props: UsersProps) {
   return (
     <SafeSuspense fallback="Loading...">
       <Users {...props} />
     </SafeSuspense>
   );
 }
 
+interface BridgeWrapperProps {
+  context: NextPageContext,
+  preloadQuery: (
+    users_pageQuery: GraphQLTaggedNode,
+    options: Record<string, unknown>
+  ) => unknown
+}
+
 export default withPageBridge({
   PageComponent: UsersWrapper,
-  getInitialProps: async ({ context, preloadQuery }) => {
-    /**
-     * If we needed to redirect from this page, we can return an object
-     * like the one below. If the returned props have a `redirect` key of type
-     * `{ destination: string, permanent?: boolean | number }`, we'll redirect the user
-     * to the destination. `permanent` defaults to `false`. If permanent is `false`,
-     * we'll treat it as a 307 redirect. If it's true, it'll be a 308. You can also pass
-     * `permanent` a redirect status code to set it directly.
-     */
+  getInitialProps: async ({ context, preloadQuery }: BridgeWrapperProps ) => {
+
+    // If we needed to redirect from this page, we can return an object
```

Comment:
I changed the comment style away from TSDoc because VSCode was attaching the comment to the type information of the first uncommented line below. I believe the intention is for the comment to reference the commented out code and not `usersQuery` on line 83.